webapp/channels/src/components/profile_popover/components/profile_popover_user_name.tsx

```diff
@@ -9,19 +9,22 @@ type Props = {
     username: string;
 }
 
-const UserName = ({
+const ProfilePopoverUserName = ({
     hasFullName,
     username,
 }: Props) => {
-    const userNameClass = classNames('overflow--ellipsis pb-1', {'user-profile-popover__heading': !hasFullName});
     return (
         <div
             id='userPopoverUsername'
-            className={userNameClass}
+            className={classNames({
+                'user-profile-popover__heading': !hasFullName,
+                'user-profile-popover__non-heading': hasFullName,
+            })}
+            title={username}
```

Comment:
Why add a title here?

Reply:
That is in case when name is long,and it gets ellipsis so title will show the full name